pkgs/by-name/pl/plakar/package.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, buildGoModule
+}:
+
+buildGoModule rec {
+  pname = "plakar";
+  version = "0.4.24-alpha";
+
+  src = fetchFromGitHub {
+    owner = "PlakarKorp";
+    repo = "plakar";
+    rev = "v${version}";
+    hash = "sha256-sAhBKIpPQL4n948ttICTUgvF7QbqqjYh+EdilEfu9XI=";
+  };
+
+  vendorHash = "sha256-AqhkX//8c7cc5LBBc4sjMFWPPKMjS7voM/X0lxlnHLU=";
+
+  meta = with lib; {
+    homepage = "https://github.com/PlakarKorp/plakar";
+    description = "plakar is a backup solution";
```

The description shouldn't start with the package name. Taking some inspiration from the website:

```suggestion
    description = "effortless, distributed backup solution";
```